wizard-frontend/src/components/HomepageReturning.tsx

```diff
@@ -0,0 +1,64 @@
+import { Plus, BookOpen } from "lucide-react";
+import { CardTitle } from "./ui/card";
+import Panel from "./Panel";
+import { LearningStepsReturning } from "./steps/LearningSteps";
+import type { WizardStep } from "./Wizard";
+import BoilerplateStep from "./steps/BoilerplateStep";
+import ConfigStep from "./steps/ConfigStep";
+
+const HomepageReturning = () => {
+  const titleCreateConfig = (
+    <CardTitle className="flex items-center gap-2">
+      <Plus className="h-5 w-5" /> Create Configuration
+    </CardTitle>
+  );
+  const titleLearn = (
+    <CardTitle className="flex items-center gap-2">
+      <BookOpen className="h-5 w-5" /> Learn About mirrord
+    </CardTitle>
+  );
+
+  const boilerplateStep = BoilerplateStep();
+  const configStep = ConfigStep();
+
+  const configSteps: WizardStep[] = [boilerplateStep, configStep];
```

Comment:
```suggestion
  const configSteps: WizardStep[] = [BoilerplateStep(), ConfigStep()];
```

Can't we do something like this?